app/org/sagebionetworks/bridge/play/controllers/ScheduleController.java

```diff
@@ -40,4 +40,8 @@ public Result getSchedules() throws Exception {
         return okResult(schedules);
     }
     
+    @Deprecated
+    public Result getSchedulesV1() throws Exception {
+        return getSchedulesV1();
+    }
```

Comment:
Infinite recursion!

Incidentally, it might be worthwhile to keep the integration tests as is to test the API that clients are currently calling. We can have a parallel v3 branch that runs the v3 APIs, and once we make the switch, we can delete the old v1 integration tests.